tools/createSovereignGenesis/create-sovereign-genesis.ts

```diff
@@ -197,12 +198,21 @@ async function main() {
     if (createGenesisSovereignParams.setPreMintAccount === true) {
         logger.info('Add preMintAccount');
 
-        // add preMintAccount.address & preMintAccount.balance
-        finalGenesis.genesis.push({
-            accountName: 'preMintAccount',
-            balance: BigInt(createGenesisSovereignParams.preMintAccount.balance).toString(),
-            address: createGenesisSovereignParams.preMintAccount.address,
+        // check preMintAddress is in the current genesis
+        const preMintAccountExist = finalGenesis.genesis.find(function (obj) {
+            return obj.address === createGenesisSovereignParams.preMintAccount.address;
         });
+
+        if (typeof preMintAccountExist !== 'undefined') {
+            preMintAccountExist.balance = BigInt(createGenesisSovereignParams.preMintAccount.balance).toString();
+        } else {
+            // add preMintAccount.address & preMintAccount.balance
+            finalGenesis.genesis.push({
```

Comment:
@krlosMata Here you push an entry to the genesis without `nonce`, which breaks later on the `_formatGeth` helper function when trying to parse the `contract.nonce` to a BigInt (from undefined).

Reply:
fixed in latest commit

Reply:
check if the account has code, it should not have, in case exists

Reply:
added sanity check